R/mod_roc.R

```diff
@@ -1902,16 +1914,21 @@ get_histo_data <- function(ds) {
 #' `r doc_templates()[["subset_data_return"]]`
 #'
 #' @param ds `[data.frame()]`
+#'
 #' A dataframe
 #'
+#' @param quantile_type `[integer(1)]`
+#'
+#' Quantile algorithm type (an integer between 1 and 9, default 7).
```

Comment:
In this internal function `quantile_type` argument does not have a default set.
```suggestion
#' Quantile algorithm type (an integer between 1 and 9).
```